spec/spec_helper.rb

```diff
@@ -1,17 +1,17 @@
 # frozen_string_literal: true
 
-require 'rspec/core'
 require 'simplecov'
-require 'ears'
-
 SimpleCov.start do
```

Comment:
like this it added coverage check only for the spec files, but not for the actual files